src/menu.js

```diff
@@ -0,0 +1,54 @@
+// Menu component
+const Menu = ({ pizzaData }) => {
+  return (
+    <main className="menu">
+      <h2>Our Menu</h2>
+
+      {/* Conditional Rendering - only render pizza list if there are pizza available (Using && operator) */}
+
+      {pizzaData.length > 0 && (
+        // {/* React Fragment - when we don't want to wrap two elements in one div */}
+        <>
+          <p>
+            Authentic italian cuisin. 6 creative dishes to choose from. All from
+            our stone own, all organic, all delicious.
+          </p>
+          <ul className="pizzas">
+            {pizzaData.map((pizza) => (
+              <Pizza pizzaObj={pizza} key={pizza.name} />
+            ))}
+          </ul>
+        </>
+      )}
+
+      {/* <Pizza
+        photoName={pizzaData[0].photoName}
+        pizzaName={pizzaData[0].name}
+        ingredients={pizzaData[0].ingredients}
+        price={pizzaData[0].price}
+      /> */}
+    </main>
+  );
+};
+
+// single pizza list item component
+const Pizza = ({ pizzaObj }) => {
```

Comment:
although I asked to remove the key prop, but if you had added there and not extracted it here, why ?

key don't work like that.

always create separate file for new comonents